tests/commands/test_command_ingest.py

```diff
@@ -15,106 +13,13 @@ def runner():
     return click.testing.CliRunner()
 
 
-def test_help(runner):
```

Comment:
I am going to add this test back in.

The other tests here look like they were removed since they were redundant.